nrcemt/qeels/test/test_peak_detection.py

```diff
@@ -358,16 +363,12 @@ def test_peak_detection():
     returned_results, returned_image = peak_detection(
         plasmon_array, width_array,
         results_array, detect_array,
-        spectrogram
+        spectrogram, 15
     )
 
-    # upper 0.46826
-    # lower 0.43952
-    # ev 0.051474
-
-    assert percent_error(0.051474, returned_results[0]) <= 3
-    assert percent_error(0.46826, returned_results[1]) <= 3
-    assert percent_error(0.43952, returned_results[2]) <= 3
+    assert percent_error(0.051474, returned_results[0]) <= 5
+    assert percent_error(0.46826, returned_results[1]) <= 5
+    assert percent_error(0.43952, returned_results[2]) <= 5
 
```

Comment:
what caused the results error to increase?

Reply:
it hasnt, idk why i changed it